scripts/common.py

```diff
@@ -170,6 +170,7 @@ def update_args(self, args=None):
 
         env_target_dir = os.getenv('CARGO_TARGET_DIR')
         self.TARGET_DIR = "{}/".format(build_type)
+        assert self.TARGET_DIR is not None  # for mypy
         if env_target_dir:
```

Comment:
See the above comment aboue `if env_suffix is not None`.  It seems to work this way here.

Reply:
Obviated by simpler cleanups.